### PR TITLE
Sync database with cached entries

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(2) Clean up job for local-storage should remove db entries](https://trello.com/c/2CRtfOg0/4361-2-clean-up-job-for-local-storage-should-remove-db-entries)

**Comment by andrecardoso** [comment]
@tejohnso sounds like a good idea, but where is the lstatSync call?

**Comment by tejohnso** [comment]
In *platform.fileExists*

File: src/db/lokijs/database.js [hunk 1]
```diff
@@ -19,7 +20,45 @@ const initCollections = () => {
...
+    .filter(entry => {
+      const pathInCache = fileSystem.getPathInCache(entry.filePath, entry.version);
+      return cached.indexOf(pathInCache) < 0;
+    })
+    .forEach(entry => {
+      log.file(JSON.stringify(entry), "File not found in cache dir. Removing it from database");
+      metadata.remove(entry)
+    });
+  })
+  .catch(err => log.file(err.message, "Error when reading cache dir to sync metadata database"));
```
Comment (santiagonoguez) [inline src/db/lokijs/database.js]: why aren't you treating this as a log.error() or a log.warning() ?

Comment (andrecardoso) [reply src/db/lokijs/database.js]: I added it to fix some tests that were failing on CI. We need to change but that requires some changes in the global.log object. Which one do you recommend `error` or `warning`?

Comment (santiagonoguez) [reply src/db/lokijs/database.js]: as it's not something critical I think warning would do. Just remember log.warning() receives just one argument and needs latest common-display-module

Comment (andrecardoso) [reply src/db/lokijs/database.js]: Thanks!

**Pull request merged**